metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -1309,3 +1323,49 @@ def create_bigquery_temp_table(
         return bigquery_temp_table
     finally:
         raw_connection.close()
+
+
+def _get_columns_to_ignore_sampling(
+    dataset_name: str, columns: List[str], tagsToIgnore: Optional[List[str]]
+) -> Tuple[bool, List[str]]:
+    logger.info("Collecting columns to ignore for sampling")
+
+    # Initialize variables
+    ignore_table: bool = False
+    columns_to_ignore: List[str] = []
+
+    if not tagsToIgnore:
+        return ignore_table, columns_to_ignore
+
+    # Generate dataset urn
+    dataset_urn = mce_builder.make_dataset_urn(
+        name=dataset_name, platform="hive", env="PROD"
+    )
```

Comment:
Why is `metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py` has hard-coded for hive and `PROD`?

Reply:
What happens in case some other source is used here? What happens in case someone uses a different env?

Reply:
made this generic, and not setting the `hive` hardcoded, have tested with `MySQL` and ignore sampling is working with MySQL as well